### ISSUE TITLE
Doc bug: Link color of types in docstrings is an inactive color instead of matching signatures

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/arcade#2286]
## Bug Report

This is a teething issue for the new theme customization on the tip of development.

### Actual behavior:
Type reference links are rendered as gray instead of matching the blue used for types in signatures.

#### Live Example:
The `SpriteList` here is gray:
![image](https://github.com/user-attachments/assets/290d4918-2478-423f-8859-09ed29a1b9f4)
![image](https://github.com/user-attachments/assets/0eede563-be43-44cc-a831-4cf3641a7583)

#### Second local example:
``str`` is a gray color, which looks inactive / unclickable:
![image](https://github.com/user-attachments/assets/0fc0236c-269e-491e-b67b-079f582910bd)

### Expected behavior:
``str`` uses the same color as `Any` and `bool` in the signature.

Note that type cross-reference links matched color with the signatures on the 2.6 doc:
![image](https://github.com/user-attachments/assets/0b928329-30b0-41ab-9f58-484e45b8956b)

For evidence:
1. Got to https://api.arcade.academy/en/latest/api/sprites.html#arcade-sprite
2. Scroll down to the texture field as in the image

### Steps to reproduce/example code:
1. Go to https://api.arcade.academy/en/development/api_docs/api/sprites.html#arcade.Sprite
4. Look at the top-level docstring
5. Hover over the `SpriteList` link here: 
    ![image](https://github.com/user-attachments/assets/290d4918-2478-423f-8859-09ed29a1b9f4)
6. Observe that it is indeed a clickable type reference link
7. Move the mouse to the lower right-hand corner to the dark / light toggle here:
    ![image](https://github.com/user-attachments/assets/f09b4306-8415-4137-a59c-46fb53e58bc1)
8. Observe the color is still gray:
    ![image](https://github.com/user-attachments/assets/3e17cc1f-a2ce-4be0-a3a8-9a20593f27c1)

**Comment by einarf** [comment]
I don't think we should change this. It has always been the standard with sphinx/rtd. Making it different from everywhere else I doubt is a good idea.

Be free to reopen if there is a more specific plan on how to change this.
